go/ql/src/Security/CWE-347/MissingJwtSignatureCheck.qhelp

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+     <overview>
+          <p>Applications decoding a JSON Web Token (JWT) may be vulnerable when the
+               signature is not correctly verified.</p>
+     </overview>
+     <recommendation>
+          <p>Always verify the signature by using the appropriate methods provided by the JWT
+               library, or use a library that verifies it by default.</p>
+     </recommendation>
+     <example>
+          <p>The following example shows a case where a JWT is parsed without verifying the
+               signature.</p>
+          <sample src="MissingJwtSignatureCheckBad.go" />
+          <p>In the example below, the appropriate function for parsing a JWT
+               and verifying its signature is used.</p>
+          <sample src="MissingJwtSignatureCheckGood.go" />
```

Comment:
```suggestion
          <p>The following (bad) example shows a case where a JWT is parsed without verifying the
               signature.</p>
          <sample src="MissingJwtSignatureCheckBad.go" />
          <p>The following (good) example uses the appropriate function for parsing a JWT
               and verifying its signature.</p>
          <sample src="MissingJwtSignatureCheckGood.go" />
```

Minor: this would make the content consistent with the way we present bad/good examples. If you're able to annotate the code as well, that'd be great, but this is probably enough!

Reply:
Thanks, applied! The code is already annotated with GOOD/BAD:

https://github.com/github/codeql/blob/8b8cebd5990de27d5105c47d824b23d0588e5509/go/ql/src/Security/CWE-347/MissingJwtSignatureCheckBad.go#L13

https://github.com/github/codeql/blob/8b8cebd5990de27d5105c47d824b23d0588e5509/go/ql/src/Security/CWE-347/MissingJwtSignatureCheckGood.go#L13

Unless you're referring to something else?

Reply:
Ah no, thanks! I didn't think to look inside the function 😅 (I'd expected it at the top). This looks fine to me